app/assets/javascripts/controllers/header_controller.js.coffee

```diff
@@ -1,12 +1,5 @@
 Blog = angular.module("Blog")
 
 Blog.controller('HeaderController', ($scope, Notification) ->
```

Comment:
do we need this controller anymore? everything was moved to BaseController so this can probably be removed